quickstarts/host-inventory-quickstart.yaml

```
@@ -21,9 +21,29 @@ spec:
 
 
   tasks:
-    - description: >-
+    - description: |-
 
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. Go to **Host inventory**.
+
+          b. Select **Configure host inventory settings**.
+
+        1. In the 'Configure host inventory settings' modal that opens, 
```

I think you could combine step a with this task so it is formatted more like a task:
```suggestion
        1. Enter the required values for Database storage, System Storage, and Image Storage in the modal that opens. 
```

---

Also Database storage, System Storage, and Image Storage should be bolded based on https://docs.openshift.com/container-platform/4.11/web_console/creating-quick-start-tutorials.html#quick-start-content-guidelines-formatting-UI-elements_creating-quick-start-tutorials